README.md

```diff
@@ -8,7 +8,7 @@ Development Setup
 -----------------
 
   1. Install: `npm install`
-  2. Run the webserver: `gulp webserver`
+  2. Run the webserver: `./node_modules/.bin/gulp`
```

Comment:
The preference is for it to not be installed globally at all and left local. Then map a `start` script to gulp:

```json
"scripts": {
  "start": "gulp"
}
```

So you start the web server with `npm start`.